<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AjouterPersonnelDialog</class>
 <widget class="QDialog" name="AjouterPersonnelDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ajouter Personnel</string>
  </property>
  <widget class="QPushButton" name="ajouterButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>410</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Ajouter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="annulerButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>410</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
  </widget>
  <widget class="QLabel" name="nomLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Nom:</string>
   </property>
  </widget>
  <widget class="QLabel" name="prenomLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Prenom:</string>
   </property>
  </widget>
  <widget class="QLabel" name="typeLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Type:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nomLineEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>401</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>xxx</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="prenomLineEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>150</y>
     <width>401</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>xxx</string>
   </property>
  </widget>
  <widget class="QComboBox" name="typeComboBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>250</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>annulerButton</sender>
   <signal>clicked()</signal>
   <receiver>AjouterPersonnelDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
